### PR TITLE
Dependencies updated

### PR DESCRIPTION
**Pull request by BBeretta**
Updated dependencies for the file to work correctly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,19 +10,43 @@ A realization of the popular trading card game - yugioh on web using React + Nod
...
+2. Change directories to `./yugioh_web_server`
+3. Command: `npm install`
+4. `git clone https://github.com/BBeretta/yugioh_web.git`
+4. Change directories to `./yugioh_web`
+5. Command: `npm install`
+
+## Running Server
+1. Open two terminal windows up and run the following in each of them
+2. Change directories to `./yugioh_web_server`
+3. `node index.js`
```
Comment (rickypeng99) [inline README.md]: Thanks for the PR. Are you sure we need to run the server twice? I have an impression that the server is centralized

**Review by rickypeng99 — COMMENTED** [review]
Thanks for the PR. I have left one comment in README.md

File: README.md (hunk 1 above)
Comment (BBeretta) [reply README.md]: Hey, sorry! 
My mistake! Copy and Paste fails :P

Comment (rickypeng99) [reply README.md]: Np, great work!

**Pull request merged**